stanford_media.module

```diff
@@ -241,3 +242,47 @@ function stanford_media_entity_access(EntityInterface $entity, $operation, Accou
 
   return AccessResult::neutral();
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ */
+function stanford_media_file_insert(FileInterface $file) {
+  $extension = pathinfo($file->getFileUri(), PATHINFO_EXTENSION);
+  $file_path = \Drupal::service('file_system')->realpath($file->getFileUri());
+  if (
+    !in_array($extension, ['jpg', 'jpeg', 'png']) ||
```

Comment:
does it matter that .gif is not included here?

Reply:
I was worried that an animated gif would break when stripping metadata. Also I doubt people are uploading a gif that they took with their Iphones.

Reply:
that sounds perfectly reasonable.